core/src/operations.rs

```diff
@@ -257,28 +257,11 @@ impl LockOp {
 
             // update file readonly flag for requested paths as appropriate
             // See the GIT_LFS_SET_LOCKABLE_READONLY section at https://www.mankier.com/5/git-lfs-config#List_of_Options-Other_settings
-            let mut should_set_read_flag = true; // this flag defaults to true if it is left unspecified
+            let mut should_set_read_flag = false; // this flag defaults to false if it is left unspecified
 
-            match env::var("GIT_LFS_SET_LOCKABLE_READONLY") {
-                Ok(env_str) => {
-                    if let Ok(env_bool) = git::parse_bool_string(&env_str) {
-                        should_set_read_flag = env_bool;
-                    }
-                }
-                Err(_) => {
-                    let output = self
-                        .git_client
-                        .run_and_collect_output(
-                            &["config", "--get", "lfs.setlockablereadonly"],

```

Comment:
Why did this code get deleted? It's checking the setting for the lockablereadonly flag.

Reply:
we ensure this setting is off on startup, so not sure it's worth checking? i could leave it in for safety but if our code doesn't intend to honor it....

Reply:
ok yeah that makes sense